packages/formation-react/stories/sidenav.stories.mdx

```diff
@@ -0,0 +1,122 @@
+import { Canvas, Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Library/Sidenav" />
+
+# Sidenav
+
+Hierarchical, vertical navigation to place at the side of a page.
+
+<Canvas>
+  <>
+    <div>
+      <button type="button" class="va-btn-sidebarnav-trigger" aria-controls="va-detailpage-sidebar">
```

Comment:
This is showing up on mobile, but when I click it nothing happens. I opened the inspector tab to see if the [formation js was changing the DOM](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52cc2dda389039da19855899e25ea524bbe195c7/packages/formation/js/sidenav.js#L98-L102) and I didn't see anything happen, so I'm not sure where things are going wrong.

It looks like things are set up correctly, with `aria-controls` matching the `id` of the `<nav>` node. I wonder if the formation JS isn't loaded on this page since there technically aren't any stories, or if [`addSidenavListeners`](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/52cc2dda389039da19855899e25ea524bbe195c7/packages/formation/js/index.js#L18) is being called before everything gets rendered.

Reply:
This goes back to the dumb `formation.js` thing. Like on the collapsible panel static markup, refresh the page and don't navigate away from it, and it'll work as expected. :roll_eyes:

Reply:
That's...strange. Anyway, this PR seems good to go in my opinion.